kernel/libs/aster-bigtcp/Cargo.toml

```diff
@@ -8,7 +8,7 @@ edition = "2021"
 [dependencies]
 keyable-arc = { path = "../keyable-arc" }
 ostd = { path = "../../../ostd" }
-smoltcp = { git = "https://github.com/smoltcp-rs/smoltcp", rev = "469dccd", default-features = false, features = [
+smoltcp = { git = "https://github.com/lrh2000/smoltcp", rev = "37716bf", default-features = false, features = [
     "alloc",
```

Comment:
Could you please submit your patch for smoltcp to our forked repository at https://github.com/asterinas/smoltcp? I’ve granted your account admin access to the repo, so you can push your code directly.

Reply:
Thanks! Will do it later. I'll put Asterinas-specific commits in a `for-asterinas` branch, just like [`lrh2000/smoltcp/tree/for-asterinas`](https://github.com/lrh2000/smoltcp/tree/for-asterinas).